packages/realm/src/Realm.ts

```diff
@@ -547,6 +547,9 @@ export class Realm {
         encryptionKey: Realm.determineEncryptionKey(config.encryptionKey),
         syncConfig: config.sync ? toBindingSyncConfig(config.sync) : undefined,
         forceSyncHistory: config.openSyncedRealmLocally,
+        automaticallyHandleBacklinksInMigrations: config.migrationOptions
+          ? config.migrationOptions.resolveEmbeddedConstraints === true
+          : false,
```

Comment:
One alternative. (Feel free to choose whichever. If going with the one below, double check linting is happy 🙂)
```suggestion
        automaticallyHandleBacklinksInMigrations: config.migrationOptions?.resolveEmbeddedConstraints ?? false,
```